src/test/java/com/wikia/webdriver/elements/mercury/pages/InfoboxBuilderPage.java

```diff
@@ -313,14 +311,22 @@ public InfoboxBuilderPage verifyScrollbarIsVisible() {
     return this;
   }
 
-  public InfoboxBuilderPage verifyTooltipOnHover() {
+  public InfoboxBuilderPage hoverMouseOverComponent(int index){
     wait.forElementVisible(component.get(0));
-    builder.moveToElement(component.get(0)).perform();
-    Assertion.assertTrue(tooltip.isDisplayed());
+    builder.moveToElement(component.get(0)).pause(500).perform();
 
     return this;
   }
 
+  public boolean isTooltipVisible() {
+    try{
+      wait.forElementVisible(tooltip);
+      return true;
+    }catch (TimeoutException e){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)